### PR TITLE
Refactor contextmenu creation

### PR DESCRIPTION
**Pull request by OPersian**
Refactored menu labels update
Changed playback rate assignment
Moved hide/show of submenu to css

### CODE REVIEW

**Comment by z4y4ts** [comment]
@polesye please take a look. I've had some concerns regarding context menu implementation IIRC.

File: video_xblock/static/js/player-context-menu.js [hunk 1]
```diff
@@ -16,82 +19,93 @@ domReady(function() {
...
   var player = videojs('{{ video_player_id }}');
 
+  /**
+   * Create elements of nested context submenu.
+   */
   function createNestedContextSubMenu(e) {
     var target = e.target;
 
     if (target.matches("li.vjs-menu-item")
         && target.innerText == player.contextmenuUI.content[3].label
```
Comment (polesye) [inline video_xblock/static/js/player-context-menu.js]: Can we fix `content[3]` here?

File: video_xblock/static/js/player-context-menu.js [hunk 2]
```diff
@@ -16,82 +19,93 @@ domReady(function() {
...
+        (function() {
+          var liSubMenu = document.createElement('li');
+          var playbackRate = playbackRates[i];
+          liSubMenu.className = 'vjs-submenu-item';
+          liSubMenu.innerHTML = playbackRate + 'x';
+          ulSubMenu.appendChild(liSubMenu);
+          liSubMenu.onclick = function() {
+            player.playbackRate(parseFloat(playbackRate));
+          }
+        })();
```
Comment (polesye) [inline video_xblock/static/js/player-context-menu.js]: it's better to use DocumentFragment in bulk DOM elements generations.
`forEach` method can also help you to simply the code.

Comment (polesye) [inline video_xblock/static/js/player-context-menu.js]: https://developer.mozilla.org/en/docs/Web/API/Document/createDocumentFragment

**Pull request merged**